plugin/web/extensions/infobox/core/components/editPanel/FieldsEditor.tsx

```diff
@@ -0,0 +1,109 @@
+// import TEST_FIELDS from "@web/extensions/infobox/TEST_FIELDS";
+import type { Field } from "@web/extensions/infobox/types";
+import { Button } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+import { useCallback } from "react";
+import { DndProvider } from "react-dnd";
+import { HTML5Backend } from "react-dnd-html5-backend";
+
+import FieldItem from "./FieldItem";
+
+type Props = {
+  fields: Field[];
+  isSaving: boolean;
+  onFieldCheckChange: (e: any) => void;
+  onFieldTitleChange: (e: any) => void;
+  onFieldMove: (dragIndex: number, hoverIndex: number) => void;
+  saveTemplate: () => void;
+};
+
+const FieldsEditor: React.FC<Props> = ({
+  fields,
+  isSaving,
+  onFieldCheckChange,
+  onFieldTitleChange,
+  onFieldMove,
+  saveTemplate,
+}) => {
+  const onSave = useCallback(() => {
+    saveTemplate();
+  }, [saveTemplate]);
```

Comment:
Are these kind of callbacks really necessary? 
I think this just makes React do more work for no real benefit since there is no additional calculations other than the one function.
If I am wrong, let me know. But I think callbacks like this will just add overhead

Reply:
Yes you're right. There was some other logic here, after some refactor the save is implemented in parent hooks so it should be removed here.